workspaces/optic-engine/src/projections/spectacle/endpoints.rs

```diff
@@ -123,7 +123,10 @@ impl AggregateEvent<EndpointsProjection> for EndpointEvent {
         }
       }
       EndpointEvent::RequestRemoved(e) => {
-        projection.without_request(e.request_id);
+        projection.without_request(&e.request_id);
```

Comment:
We could also change the other consumers here to pass in references instead of `clone()` - where there's a with_creation_history, etc